apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/ChecklistUtils.tsx

```diff
@@ -93,113 +95,134 @@ export type ChecklistStatus = {
 export const CHECKLIST_STATUSES: ChecklistStatus = {
   ServiceKey: {
     success: {
-      icon: getSuccessIcon('Service account is correctly installed'),
+      icon: getSuccessIcon(''),
       title: CHECKLIST_NAMES.serviceAccount,
-      description: 'Success!',
+      description: 'Service account key is valid',
       disabled: false,
     },
     invalid: {
       icon: getErrorIcon(
-        'Failed to connect to your service account. Re-install your service account key if the problem persists'
+        `We couldn't connect to Google Analytics APIs using the service account key provided. Try re-installing your previous service account key or create and install a new key`
```

Comment:
i think the addition of `previous` here makes this weird. suggest removing or saying `current` instead – otherwise looks great

Reply:
Agreed